### PR TITLE
Create woocommerce

### PR DESCRIPTION
**Pull request by powermyanalytics**

### CODE REVIEW

File: organizations/woocommerce [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (pfrisella) [inline organizations/woocommerce:1]: file is missing .json extension

**Review by pfrisella — CHANGES_REQUESTED** [review]
There is no source file, is that intended?

**Comment by powermyanalytics** [comment]
Can you please add ".json" at the end - my bad...

**Comment by powermyanalytics** [comment]
Hi Pete,

 

Can we get these published?  We have been waiting over 2 weeks to submit WooCommerce to the gallery.  I resubmitted the Source and Organization for WooCommerce a couple days ago to try to get those going again.

 

Cheers

 

Morgan

 

  <https://www.powermyanalytics.com/images/powermyanalytics2.png> 

 <http://www.powermyanalytics.com/> http://www.powermyanalytics.com/

Data Connectors + Call Tracking

(800) 746-3083, x701

**Comment by pfrisella** [comment]
Resolved with another PR

**Pull request closed without merging**